### PR TITLE
feat(vacancy): Vacancy feature - OEL-1454

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/compositions/bcl-vacancy/vacancy.story.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+import dataListing from "@openeuropa/bcl-vacancy/data/data--listing";
+import dataContent from "@openeuropa/bcl-vacancy/data/data";
+import dataSidebarDetails from "@openeuropa/bcl-vacancy/data/data--sidebar-details";
+import detailsPage from "@openeuropa/bcl-vacancy/vacancy.html.twig";
+
+const header =
+  layout[`header_${process.env.STORYBOOK_THEME}`] || layout.headerSimple;
+
+if (isChromatic()) {
```
Comment (planctus) [inline src/compositions/bcl-vacancy/vacancy.story.js:25]: can you do the same also when the env variable is set..?

File: src/compositions/bcl-vacancy/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@openeuropa/bcl-vacancy",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "0.22.0",
+  "description": "OE - BCL Vacancy",
+  "publishConfig": {
+    "access": "public"
```
Comment (planctus) [inline src/compositions/bcl-vacancy/package.json:8]: does it have to be public this one..?

Comment (tibi2303) [reply src/compositions/bcl-vacancy/package.json]: made it private

File: src/compositions/bcl-vacancy/vacancy.story.js (hunk 1 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/vacancy.story.js]: yes, also updated the chromatic functions, it was not working

File: src/compositions/bcl-vacancy/vacancy.story.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  decorators: [withDesign],
+  parameters: {
+    layout: "fullscreen",
+    controls: {
+      disable: true,
+    },
+    badges: ["stable"],
+    badgesConfig: {
+      stable: {
+        title: "v1.1",
```
Comment (planctus) [inline src/compositions/bcl-vacancy/vacancy.story.js:101]: but we are introducing this feature as new...why is it already 1.1..? :) Wasn't this an internal versioning of the single features.?

File: src/compositions/bcl-vacancy/vacancy.test.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+const renderDetails = (params) =>
+  renderTwigFileAsNode(detailsTemplate, params, true);
+
+describe("OE - Vacancy", () => {
+  test("listing renders correctly", () => {
+    expect.assertions(1);
+    return expect(renderListing(demoListing)).resolves.toMatchSnapshot();
+  });
+
+  test("details (closed) renders correctly", () => {
```
Comment (planctus) [inline src/compositions/bcl-vacancy/vacancy.test.js:93]: guess that this title is wrong..

File: src/compositions/bcl-vacancy/vacancy.story.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    },
+  },
+  pagination: pagination,
+  filter_button: filterButton,
+  sort_select: sortSelect,
+};
+
+export default {
+  title: "Features/Vacancy",
+  decorators: [withDesign],
```
Comment (planctus) [inline src/compositions/bcl-vacancy/vacancy.story.js:92]: don't we have withCode also, now..?

File: src/compositions/bcl-vacancy/vacancy.story.js (hunk 3 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/vacancy.story.js:101]: Talked with romain and we could already implement the 1.1 version.

File: src/compositions/bcl-vacancy/package.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@openeuropa/bcl-vacancy",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "0.22.0",
+  "description": "OE - BCL Vacancy",
+  "private": true,
+  "repository": {
```
Comment (planctus) [inline src/compositions/bcl-vacancy/package.json:8]: we miss some dependencies here..button, description-list, file, modal, base-templates

File: src/compositions/bcl-vacancy/.npmignore [hunk 7]
```diff
@@ -0,0 +1,2 @@
+__snapshots__
```
Comment (planctus) [inline src/compositions/bcl-vacancy/.npmignore:1]: now this is useless

File: src/compositions/bcl-vacancy/vacancy.story.js (hunk 5 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/vacancy.story.js]: yes, indeed, i added.

File: src/compositions/bcl-vacancy/package.json (hunk 6 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/package.json:8]: added the dependencies, but i think in some cases we are missing those, ex multilingual
image

File: src/compositions/bcl-vacancy/.npmignore (hunk 7 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/.npmignore]: deleted it

File: src/compositions/bcl-vacancy/package.json (hunk 6 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/package.json:8]: added the dependencies.

File: src/compositions/bcl-vacancy/vacancy.test.js (hunk 4 above)
Comment (tibi2303) [reply src/compositions/bcl-vacancy/vacancy.test.js]: done

**Review by planctus — APPROVED** [review]

**Pull request merged**